scenarios/kubernetes_e2e.py

```diff
@@ -103,7 +103,13 @@ def check_env(env, *cmd):
     for key, value in sorted(env.items()):
         print >>sys.stderr, '%s=%s' % (key, value)
     print >>sys.stderr, 'Run:', cmd
-    subprocess.check_call(cmd, env=env)
+    proc = subprocess.Popen(cmd, env=env, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
we should consider updating `check_output` to capture stderr instead and use it here. deferring to @krzyzacy on this one.